packages/contracts/src/RelayHub.sol

```diff
@@ -180,84 +187,84 @@ contract RelayHub is IRelayHub, Ownable {
         );
 
         (vars.gasAndDataLimits, vars.maxPossibleGas) =
-             verifyGasAndDataLimits(maxAcceptanceBudget, relayRequest, vars.initialGasLeft);
+        verifyGasAndDataLimits(maxAcceptanceBudget, relayRequest, vars.initialGasLeft);
 
-        RelayHubValidator.verifyTransactionPacking(relayRequest,signature,approvalData);
+        RelayHubValidator.verifyTransactionPacking(relayRequest, signature, approvalData);
 
-    {
+        {
```

Comment:
Please avoid this sweeping newline change until we perform the three-way merge

Reply:
again. its autoformat...